src/main/java/resources/TwitterRequestResource.java

```diff
@@ -4,12 +4,14 @@
 import handlers.ExceptionHandler;
 import model.StatusList;
 import model.Message;
+import configuration.Configuration;
 
 import twitter4j.*;
 
 import javax.ws.rs.*;
```

Comment:
in general we prefer exact imports instead of `*`.  

`HINT`: there is a setting in intellij that can help with this.